packages/nc-gui/composables/useViewData.ts

```diff
@@ -79,8 +82,8 @@ export function useViewData(
   })
 
   const queryParams = computed(() => ({
-    offset: ((paginationData.value?.page ?? 0) - 1) * (paginationData.value?.pageSize ?? 25),
-    limit: paginationData.value?.pageSize ?? 25,
+    offset: ((paginationData.value?.page ?? 0) - 1) * (paginationData.value?.pageSize ?? appInfoDefaultLimit),
+    limit: paginationData.value?.pageSize ?? appInfoDefaultLimit,
```

Comment:
The `?` should be unnecessary here as well, if the typing here is correct `paginationData` cannot be undefined.